api/rs/slint/lib.rs

```diff
@@ -443,3 +443,13 @@ mod compile_fail_tests;
 
 #[cfg(doc)]
 pub mod docs;
+
+#[cfg(feature = "unstable-wgpu-24")]
+pub mod wgpu_24 {
+    //! WGPU 24.x specific types and re-exports.
```

Comment:
I feel this module documentation is an opportunity to expand on how to use it with some code sample on how to select the right backend using the renderer selector, and perhaps add some .toml snippet on how to enable the feature and a link to  `crate::docs::cargo_features#backends`

Reply:
You're totally right. I'll extend this and add an example snippet.